lib/manticore/client.rb

```diff
@@ -680,8 +681,10 @@ def ssl_socket_factory_from_options(ssl_options)
         verifier = DefaultHostnameVerifier.new
       when :strict # compatibility
         verifier = SSLConnectionSocketFactory::STRICT_HOSTNAME_VERIFIER
+      when :noop
+        verifier = NoopHostnameVerifier::INSTANCE
       else
-        raise "Invalid value for :verify. Valid values are (:default, :browser, :none)"
+        raise "Invalid value for :verify. Valid values are (:default, :browser, :noop, :none)"
```

Comment:
this is simply confusing, I think the other values were trying to help the user understand.

but now having `:noop, :none` just adds confusion to the mix, guess we could accept `:noop` internally but maybe as undocumented or at least not shown here...

Reply:
Thanks for reviewing, I agree,`:noop` is indeed confusing, I ended up using it to keep it similar to the Apache's verifier. Other names that came to my mind are`:certificate` and `:ca`. Do you have any suggestion?